parser-typechecker/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -45,7 +45,7 @@ data Input
     -- clone w/o merge, error if would clobber
     = ForkLocalBranchI (Either ShortBranchHash Path') Path'
     -- merge first causal into destination
-    | MergeLocalBranchI Path' Path'
+    | MergeLocalBranchI Path' Path' Branch.MergeMode
```

Comment:
Not a new command, just another argument to the existing `MergeLocalBranchI` command. I could imagine adding other modes - `Theirs`, `Ours`, etc.